develop/events.rst

```diff
@@ -112,3 +112,72 @@ Regular expression triggered events
 
 The :class:`supybot.callbacks.PluginRegexp` class provides some utilities
 for creating plugins that act on regular expression matching.
+
+Reacting to URLs
+----------------
+
+Using Limnoria's ``urlSnarfer`` will avoid loops with other bots and ignore private messages.
```

Comment:
```suggestion
Using :func:`supybot.commands.urlSnarfer` will avoid loops with other bots and ignore private messages.
```

Reply:
IMO you should also add a brief (one-line) description of what this example does: it creates a new snarfer for example.com, gated behind a config variable `plugins.examplesnarfer.enabled`

Reply:
I'd also start the introduction with "As a special case of `PluginRegexp`, `urlSnarfer` can be used to act on people sending URLs."